script/grblcon.py

```diff
@@ -87,7 +116,15 @@ def __str__(self):
                 self.work_positions.c
             ),
             "Line: {}".format(self.line),
-            "Limit Flags: {}".format(self.limit_flags)
+            "Limit Flags: {}".format(self.limit_flags),
+            "ADCs: {}, {}, {}, {}, {}, {}".format(
+                self.adcs.x,
+                self.adcs.y,
+                self.adcs.z,
```

Comment:
It seems a little silly to separate all these variables out if the only place they're used has them all glommed together again. Is there something in here that can be simplified?